peachjam/js/components/DocumentSearch/ResultSnippet.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div
+    ref="snippet"
+    class="result-snippet"
+  />
+</template>
+
+<script>
+export default {
+  name: 'ResultSnippet',
+  props: {
+    node: {
+      type: HTMLElement,
+      required: true
+    }
+  },
+  watch: {
+    node (newNode) {
+      this.setHTML(newNode);
+    }
+  },
+  mounted () {
+    this.setHTML(this.node);
+  },
+  methods: {
+    setHTML (node) {
+      this.$refs.snippet.innerHTML = '';
+      // Anchor clean up
+      node.querySelectorAll('a').forEach(node => {
+        const parent = node.parentNode;
+        while (node.firstChild) parent.insertBefore(node.firstChild, node);
+        parent.removeChild(node);
+      });
+      // Inline style clean up
+      node.removeAttribute('style');
+      node.querySelectorAll('[style]').forEach(node => node.removeAttribute('style'));
+      this.$refs.snippet.appendChild(node);
+    }
+  }
+};
+</script>
+
+<style>
+.result-snippet * {
+  font-size: 16px !important;
```

Comment:
there are cases where receive heading elements (h1, h2, h3..). so we need to enforce font size

Reply:
Needs to be 14px, that's the base font size. It would be useful to move this into the global .scss file so you can rely on the relevant bootstrap sass variable.

Reply:
Alternatively, make it `font-size: inherit !important;`

Reply:
Cool, I moved that style to global.scss and set the font size to `$font-size-base`